istioctl/pkg/install/k8sversion/version_test.go

```diff
@@ -152,3 +157,36 @@ func TestExtractKubernetesVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckKubernetesVersion(t *testing.T) {
+	cases := []struct {
+		version  *version.Info
+		expected bool
+		errMsg   error
```

Comment:
nit: Can we remove errMsg as it is not needed at all?

Reply:
true, will do that. Thanks @irisdingbj